client/installation-helper.cc

```diff
@@ -195,6 +195,46 @@ step4()
     cout << "done" << endl;
 }
 
+int
```

Comment:
Since EXIT_SUCCESS is 0 which evaluates to false and EXIT_FAILURE is 1 which evaluates to true it's better to use a bool as return type and "return step5(...) ? EXIT_SUCCESS : EXIT_FAILURE" in main.